.github/workflows/tests.yml

```diff
@@ -22,12 +22,29 @@ jobs:
           - "7.3"
           - "7.4"
 
+    env:
+      MYSQL_USER: "travis"
+      MYSQL_PASSWORD: "password"
+      MYSQL_DATABASE: "zftest"
+      MYSQL_HOST: "127.0.0.1"
+
     services:
       memcache:
         image: memcached:1-alpine
         ports:
           - 11211:11211
 
+      mysql:
+        image: mysql/mysql-server:5.6
```

Comment:
could we have `5.7`?

Reply:
sure, I just wanted to test with the lowest version available. ideally should add matrix of supported versions.